src/tmx/TmxApi/CMakeFiles/tmxapi.dir/flags.make

```diff
@@ -0,0 +1,17 @@
+# CMAKE generated file: DO NOT EDIT!
+# Generated by "Unix Makefiles" Generator, CMake Version 3.10
+
+# compile C with /usr/bin/cc
+# compile CXX with /usr/bin/c++
+C_FLAGS = -fPIC  
+
+C_DEFINES = -DBOOST_ASIO_HEADER_ONLY -DBOOST_SPIRIT_THREADSAFE  -DBOOST_ALL_DYN_LINK -DSAEJ2735_SPEC=63 -Dtmxapi_EXPORTS
+
+C_INCLUDES = -isystem /home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxApi -I/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/Asn_J2735/include/asn_j2735_r63 
```

Comment:
This file contains specific username home path. Can we remove this cache file? or git ignore it?